package.json

```diff
@@ -33,8 +33,7 @@
   },
   "dependencies": {
     "angular2": "2.0.0-beta.0",
-    "es6-promise": "^3.0.2",
-    "es6-shim": "^0.34.0",
+    "core-js": "^1.2.6",
```

Comment:
I switched back to es6-promise and es6-shim so developers won't be confused when they npm install and see that angular2 has a peer dependency of these two